.github/next_release.md

```diff
@@ -1,22 +1,40 @@
 We are pleased to announce the release 3.8.0 of the GUDHI library.
 
-As a major new feature, the GUDHI library now offers Perslay, a Tensorflow model for the representations module.
+As a major new feature, the GUDHI library now offers Perslay, a Tensorflow model for the representations module, scikit-learn like interfaces for Cover Complexes, a new function to compute persistence of a function on $\mathbb{R}$ and the possibility to build a Cubical Complex as a lower-star filtration from vertices.
 
 We are now using GitHub to develop the GUDHI library, do not hesitate to [fork the GUDHI project on GitHub](https://github.com/GUDHI/gudhi-devel). From a user point of view, we recommend to download GUDHI user version (gudhi.3.X.X.tar.gz).
 
 Below is a list of changes made since GUDHI 3.7.1:
 
 - [Perslay](https://gudhi.inria.fr/python/latest/representations_tflow_itf_ref.html)
-     - a TensorFlow layer for persistence diagrams representations
+     - a TensorFlow layer for persistence diagrams representations.
+
+- [Cover Complex](https://gudhi.inria.fr/python/latest/cover_complex_sklearn_user.html)
+     - New classes to compute Mapper, Graph Induced complex and Nerves with a scikit-learn like interface.
+
+- [Persistent cohomology](https://gudhi.inria.fr/doc/latest/group__persistent__cohomology.html)
+     - New linear-time `compute_persistence_of_function_on_line`, also available though `CubicalPersistence` in Python.
+
+- [Cubical complex](https://gudhi.inria.fr/doc/latest/group__cubical__complex.html)
+     - Add possibility to build a lower-star filtration from vertices instead of top-dimensional cubes.
+     - Faster persistence computation of 1d cubical complexes.
```

Comment:
I would skip this one. The thing that really improves persistence of 1d cubical complexes is the new `compute_persistence_of_function_on_line` (that name is not pretty), but it isn't part of the C++ or Python cubical complex.